<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yumlistPackageDlg</class>
 <widget class="QDialog" name="yumlistPackageDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="search_lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="search_Btn">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
